packages/parameters/src/ExpirableValue.ts

```diff
@@ -4,10 +4,17 @@ class ExpirableValue implements ExpirableValueInterface {
   public ttl: number;
   public value: string | Uint8Array | Record<string, unknown>;
 
+  /**
+   * Creates a new cached value which will expire automatically
+   * @param value Parameter value to be cached
+   * @param maxAge Maximum number of seconds to cache the value for
+   */
   public constructor(value: string | Uint8Array | Record<string, unknown>, maxAge: number) {
     this.value = value;
-    const timeNow = new Date();
-    this.ttl = timeNow.setSeconds(timeNow.getSeconds() + maxAge);
```

Comment:
Why not simply do:
```ts
this.ttl = timeNow.setSeconds(timeNow.getSeconds() + maxAge * 1000);
```

and minimize the diff?

Reply:
I was exploring different algorithms as I tested and this is the last one I landed on before realizing the larger issue. It was a little easier for me to wrap my head around the intent.

[From the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setSeconds), `Date.setSeconds()` takes a seconds value, not a milliseconds one, so the original implementation might be right.

In my node REPL:

```javascript
> const timeNow = new Date();
undefined
> const maxAge = 20;
undefined
> timeNow.getTime()
1678466472800
> timeNow.setSeconds(timeNow.getSeconds() + maxAge);
1678466492800
> 1678466492800-1678466472800 // New value is 20000ms ahead of old value... 20 seconds
20000
```